recipes/libcurl/all/conanfile.py

```diff
@@ -259,7 +259,7 @@ def _get_configure_command_args(self):
             params.append("--without-nghttp2")
 
         if self.options.with_zlib:
-            params.append("--with-zlib={}".format(tools.unix_path(self.deps_cpp_info["zlib"].lib_paths[0])))
+            params.append("--with-zlib={}".format(tools.unix_path(self.deps_cpp_info["zlib"].rootpath)))
```

Comment:
Prefer to use pkgconf: https://github.com/curl/curl/blob/f3294d9d86e6a7915a967efff2842089b8b0d071/configure.ac#L904

Reply:
I guess it's too much, it has to be tested first, and it's not the goal of the PR, so maybe for another one